### DISCUSSION TITLE
tag search display

### DISCUSSION THREAD

**Discussion by plum** [nvim-telekasten/telekasten.nvim#348] [Q&A]
When  searchin gfor tags (Show tags)  and then selecting one  tag, upon hitting enter  a list of files is presented in a window (i.e. telescope-fzf-native.nvim") ; 

QUESTION: **Is it possible ot get a preview of each file as well?**

Here is an example of telekasten in action:
![tags_telekasten_selct_tag](https://github.com/user-attachments/assets/c38d9418-2138-4961-8e6a-61ecff534791)

ANd here is an example using :Fzflua tags   which does show a file  preview, and is how  I would like to usefully  examine reults of tag searches; However this methos relies upon ctags.

![tags_Fzflua_tags](https://github.com/user-attachments/assets/95e1befb-c6ba-478a-afac-8f4cf152e525)

Thanks, John

**Comment by lambtho12** [comment]
Are you sure it is not something related to your screen size ? When I pick a tag, I get shown the preview of the files. But if I resize the terminal window to half my screen width, Telescope only shows the search portion and not the preview one.
See below what I have when my terminal is in full width or half width of the screen...

![fullscreen](https://github.com/user-attachments/assets/a6091611-992f-4c26-b342-f1c2ff77d5ab)
![halfscreen](https://github.com/user-attachments/assets/7e818dd9-7530-490d-97a3-fb6a006ec9b0)

**Reply by SpekkleDust** [reply]
Hi,

I can see the preview too.

KindRegards
